MonoGame.Framework.Content.Pipeline/TextureImporter.cs

```diff
@@ -53,7 +53,7 @@ public class TextureImporter : ContentImporter<TextureContent>
         /// <summary>
         /// Initializes a new instance of TextureImporter.
         /// </summary>
-        public TextureImporter()
+        public TextureImporter( )
```

Comment:
Reformatting, especially that which is just a whitespace change, is not wanted in a PR.  It adds noise to a PR and hides the real changes.